src/Serilog.Sinks.MSSqlServer/Sinks/MSSqlServer/Output/AdditionalColumnDataGenerator.cs

```diff
@@ -28,14 +28,14 @@ internal class AdditionalColumnDataGenerator : IAdditionalColumnDataGenerator
                 : _columnHierarchicalPropertyValueResolver.GetPropertyValueForColumn(additionalColumn, properties);
 
             var columnName = additionalColumn.ColumnName;
-            if (property.Value == null)
+            if (!(property.Value is ScalarValue scalarValue))
             {
-                return new KeyValuePair<string, object>(columnName, DBNull.Value);
+                return new KeyValuePair<string, object>(columnName, property.Value.ToString());
             }
```

Comment:
The code used to check for `property.Value == null`, now if `property.Value not is ScalarValue` when null it'll call `property.Value.ToString()`. Is it never null? What about complex types?

Reply:
Thanks for the review. Of course there are null exceptions now. I should really run all the tests locally before pushing to a PR. At least I was able to show that we have good tests to validate a PR. 😆

Reply:
Haha, I didn't even notice the failed build, thought it was one of the code scanning things because it's prefixed "PR Validation"